resources/local/chainspec.toml.in

```diff
@@ -29,10 +29,10 @@ era_duration = '21seconds'
 # minimum height.
 minimum_era_height = 5
 # Number of slots available in validator auction.
-validator_slots = 5
+validator_slots = 7
 # Number of eras before an auction actually defines the set of validators.  If you bond with a sufficient bid in era N,
 # you will be a validator in era N + auction_delay + 1.
-auction_delay = 3
+auction_delay = 1
```

Comment:
Because of this change, the `should_load_chainspec_config_from_chainspec` and `should_get_exec_config_from_chainspec_values` tests need to be updated with the new `auction_delay` as well.

Reply:
@Fraser999 already [took care of it](https://github.com/casper-network/casper-node/pull/3465).